environment.yml

```diff
@@ -0,0 +1,9 @@
+name: particle_demo
+dependencies:
+  - python>=3.7
+  - pandas
+  - attrs
+  - pip
```

Comment:
The addition of 

```
pip
jupyterlab
```

is unnecessary as Binder's Conda base environment already supports. Though if you were interested in having a "local" Binder env this works.